tasks/request_files/request_files.py

```diff
@@ -579,9 +567,10 @@ def restore_object(
             f"{c_err}. bucket: {db_glacier_bucket_key} file: {key} Job ID: {job_id}"
         )
         raise c_err
-    if restore_result["ResponseMetadata"]["HTTPStatusCode"] == 202:
+    if restore_result["ResponseMetadata"]["HTTPStatusCode"] == 200:
```

Comment:
I would move this logic into the try block between line 562 and 563 so that the error is logged and wont't cause the Lambda to die if there are multiple files we are working through.

Reply:
The error would be raised in both cases. Logging is already handled in line 426, so I would actually vote for stripping logging from the other error case.

Reply:
Yep, I see that now. I agree on your comment to have it log in one place. In that case we could probably remove the try/except block